data/pages-content/terms.md

```diff
@@ -171,56 +162,54 @@ Organizers and/or third parties.
 The Participant accepts the EuroPython Code of Conduct (the “Code of Conduct”),
 which can be found at
 [www.europython-society.org/coc/](http://www.europython-society.org/coc/). The
-Organizers may make alterations to the Code of Conduct from time to time. Minor
-alterations are not considered contractual deviations.
+Organisers may, from time to time, make alterations to the Code of Conduct, which are not considered contractual deviations.
 
-The Code of Conduct is part of this contract and shall be observed during
-participation in the event. The Participant consents to said Code of Conduct and
+The Code of Conduct is an integral part of the agreement and shall be observed during
+participation in the Event. The Participant consents to said Code of Conduct and
 shall comply with it.
 
-### Use of the Conference Platform
+### Use of the Conference Venue / Platform
 
 The Participant will ensure or accept that:
 
 ### Use of the Venue and Online Platform
```

Comment:
I would separate the in-person and online participant rules into separate sections. There is very little overlap, so it should be clearer that way.